src/test/java/redis/clients/jedis/SSLACLJedisTest.java

```diff
@@ -23,50 +22,53 @@
  */
 public class SSLACLJedisTest {
 
+  protected static final EndpointConfig endpoint = HostAndPorts.getRedisEndpoint("standalone0-acl-tls");
+
+  protected static final EndpointConfig endpointWithDefaultUser = HostAndPorts.getRedisEndpoint("standalone0-tls");
+
   @BeforeClass
   public static void prepare() {
     // Use to check if the ACL test should be ran. ACL are available only in 6.0 and later
-    org.junit.Assume.assumeTrue("Not running ACL test on this version of Redis",
-        RedisVersionUtil.checkRedisMajorVersionNumber(6));
-
     SSLJedisTest.setupTrustStore();
+    org.junit.Assume.assumeTrue("Not running ACL test on this version of Redis",
```

Comment:
Why move the `assume` to the end of the method?

Reply:
`SSLJedisTest.setupTrustStore();` was moved higher to set up the certificate properly before we verify/connect.